mitodl/bootcamp-ecommerce
# Add Linkedin option for resume

#### What are the relevant tickets?
Fix #529

#### What's this PR do?
Add a new field to the BootcampApplication model, process the resume or linkedin url and transition the application to the next stage.

#### How should this be manually tested?
Create a bootcamp application, set the state to awaiting resume, try sending a post request with a `linkedin_url`
This should update the corresponding field on the application model, update the `resume_upload_date` field, and transition the application to next stage.
